SM/internal/services/machine.go

```diff
@@ -2,14 +2,34 @@ package services
 
 import (
 	"context"
+	"sm/internal/database/postgres"
 	"sm/internal/utils/logger"
 )
 
-func MachineNeedRepair(sp *ServicesParams, machineid int64) bool {
+func MachineNeedRepair(sp *ServicesParams, machineid int64) error {
 	err := sp.db.MachineNeedRepair(context.Background(), machineid)
 	if err != nil {
 		sp.log.Info("Failed to change status to need repair from db", logger.ErrToAttr(err))
-		return false
+		return err
 	}
-	return true
+	return nil
+}
+
+func CreateMachine(sp *ServicesParams, req Machine) error {
+	machineParams := convertCreateMachineParams(req)
+	_, err := sp.db.CreateMachine(context.Background(), machineParams)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
oftengter creation and entity initialization on the back we should return id's and other back-end-created-data to the client to provide ability use thisdata as soonaspossible